### PR TITLE
Refactor contribution page

### PR DESCRIPTION
**Pull request by ulysses-you**

### CODE REVIEW

**Comment by ulysses-you** [comment]
cc @yaooqinn about https://github.com/apache/incubator-kyuubi/issues/1563

**Comment by yaooqinn** [comment]
## Pull Request Descriptions

Providing descriptions for pull requests is highly recommended for a better code review process. Writing good pull request descriptions is a great way to help reviewers know what to expect when reviewing code. As developers, our responsibility is to fix issues or implement new features and clearly communicate the development work to reviewers. A developer can convey proposed code changes and their purposes either through detailed PR descriptions. They're also a great way to help track things that should be done for every change, such as testing, adding unit tests, and updating documentation in the `gitlog`.

Descriptions shall clearly answer:

Q1.
Q3.
Q3

**Comment by yaooqinn** [comment]
Considering adding the above content?

**Comment by ulysses-you** [comment]
yea, added

File: src/contributing.md [hunk 1]
```diff
@@ -19,64 +19,94 @@ menu:
...
+
+About the pull request, it is highly recommended providing a clear descriptions for a better code review process.
+Writing good pull request descriptions is a great way to help reviewers know what to expect when reviewing code.
+As developers, our responsibility is to fix issues or implement new features and clearly communicate the development work to reviewers.
+A developer can convey proposed code changes and their purposes either through detailed PR descriptions.
+They're also a great way to help track things that should be done for every change, such as testing, adding unit tests, and updating documentation in the git log.
+
+Descriptions shall clearly answer:
+
+Q1.
```
Comment (yaooqinn) [inline src/contributing.md:102]: what are the questions?....

**Review by yaooqinn — APPROVED** [review]

**Pull request merged**